src/machines/java/com/enderio/machines/common/blockentity/task/SpawnTask.java

```diff
@@ -20,23 +25,30 @@ public class SpawnTask extends PoweredTask{
 
     public static final int spawnTries = 10;
     private boolean complete;
-    private int energyCost = 40000;//TODO Custom Config/Json File
```

Comment:
Might be a good idea to just store the soul datum rather than storing each field separately. Might be a neat way of doing it :)

Reply:
I'm not a 100% sure on this, I found this cleaner but that's just preference